src/core/bytecodedump.c

```diff
@@ -176,7 +176,7 @@ char * MVM_bytecode_dump(MVMThreadContext *tc, MVMCompUnit *cu) {
     MVMuint32 lineloc;
     MVMuint16 op_num;
     const MVMOpInfo *op_info;
-    MVMuint32 operand_size;
```

Comment:
If `op_info->num_operands` always return more than `0` in the `for` loop on line 231 then this is not needed.